src/popover/src/Popover.js

```diff
@@ -347,7 +353,7 @@ export default class Popover extends Component {
             }}
             data-state={state}
             css={css}
-            style={style}
+            style={[style, customStyle]}
```

Comment:
I don't think array notation works? Can you replace with `Object.assign({}, style, customStyle)`?